docs/asset-store/docs/03-04-asset-metadata-service.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Asset Metadata Service
+type: Details
+---
+
+The Asset Metadata Service is an HTTP server that exposes files metadata extraction functionality. It contains a simple HTTP endpoint which accepts `multipart/form-data` forms. The service extracts front matter YAML metadata from text files in arbitrary extension.
+
+The main purpose of the service is to provide metadata extraction for Asset Store controllers. That's why it is only available inside the cluster. To use it, define `metadataWebhookService` in Asset and ClusterAsset custom resources.
+
+## Front matter metadata
+
+Front matter YAML metadata are YAML properties written at the beginning of a file, written between `---` lines. The following snippet represents an exemplary Markdown file with metadata specified:
+
+```markdown
+
+---
+title: Example document title
+description: Description of the page
+order: 3
+array:
+ - foo
+ - bar
+---
+
+# Lorem ipsum
```

Comment:
```suggestion
## Lorem ipsum
```

Reply:
In our docs we rather start with `##` headings as the title of the doc already acts as the main heading.